frontend/src/Common.js

```diff
@@ -337,16 +337,6 @@ export const ShowUploadMetadata = ({ upload }) => {
           <th>Bucket Name</th>
           <td>{upload.bucket_name}</td>
         </tr>
-        <tr>
```

Comment:
The changes for dropping `bucket_endpoint_url` should be in a separate PR given we've got a separate bug for it and (as far as I can tell) it's not tangled with the other changes.

Reply:
It's possible to split that out, but I'd like to merge that new PR first and keep the commit in this PR until we rebase. Otherwise there will be a fair number of conflicts.

Reply:
This is in #2961 now.